src/stats_tests/fisher.rs

```diff
@@ -0,0 +1,358 @@
+use crate::distribution::{Discrete, DiscreteCDF, Hypergeometric};
+use crate::StatsError;
+
+#[derive(Debug, Copy, Clone)]
```

Comment:
A brief docstring on this enum would be nice for our docs.rs, but I don't think one is needed for each of the variants.

Reply:
I approved your PR into this branch with your changes, thank you!